egov-hrms/src/main/java/org/egov/hrms/service/EmployeeService.java

```diff
@@ -337,7 +337,7 @@ public EmployeeResponse update(EmployeeRequest employeeRequest) {
 			updateUser(employee, requestInfo);
 		});
 		hrmsProducer.push(propertiesManager.getUpdateEmployeeTopic(), employeeRequest);
-
+		notificationService.sendReactivationNotification(employeeRequest);
```

Comment:
should it be in try catch? can it be async? If notification fails should the update fail?